galaxy.yml

```diff
@@ -3,12 +3,19 @@ authors:
   - Ansible Security Team (https://github.com/ansible-security)
 dependencies:
   "ansible.netcommon": ">=2.0.1"
+  "ansible.utils": '>=2.0.0'
 license_file: LICENSE
 name: deepsec
 namespace: trendmicro
 description: Ansible Security Collection for Trend Micro Deep Security
 readme: README.md
+# The URL of the originating SCM repository
 repository: https://github.com/ansible-collections/trendmicro.deepsec
+# The URL to any online docs
+documentation: https://github.com/ansible-collections/trendmicro.deepsec
```

Comment:
is it intended to provide GitHub repo link or do we have documentation available on docs.ansible.com ?